.sauce/benchmarking-config.yml

```diff
@@ -13,14 +14,25 @@ xcuitest:
 
 suites:
   - name: "High-end device"
+    passThreshold: 2
```

Comment:
Unfortunately, you can only add the `passThreshold` and the `smartRetry` for every suite and not for all suites.